include/libdnf5-cli/exception.hpp

```diff
@@ -54,6 +54,14 @@ class InsufficientPrivilegesError : public Error {
 };
 
 
+/// Exception is thrown when trying to write to a read-only system
+class ReadOnlySystemError : public Error {
```

Comment:
This enhances API and thus we should bump minor version of DNF5.

Reply:
Bumped.

Reply:
Could you move that bumping commit before a commit which adds this class? Or simply squash the two commits?

Reply:
OK, I put the bumping commit first since the "Add bootc utility functions" commit also enhances libdnf5.